File: EducationManagement.Database/Models/Subject.cs
using System;
using System.Collections.Generic;

namespace EducationManagement.Database.Models
{
    public partial class Subject
    {
        public Subject()
        {
            SubjectLevel = new HashSet<SubjectLevel>();
            TeacherSubject = new HashSet<TeacherSubject>();
        }

        public int Id { get; set; }
        public string Name { get; set; }

        public ICollection<SubjectLevel> SubjectLevel { get; set; }
        public ICollection<TeacherSubject> TeacherSubject { get; set; }
    }
}

File: EducationSystem.WEB/src/app/_models/Subject.ts
import { SubjectLevel } from "app/_models/SubjectLevel";

export class Subject {
    public Name: string;
    public Id: number;

    public SubjectLevels: SubjectLevel[];

    constructor(name?: string, id?: number) {
        this.Name = name;
        this.Id = id;
    }
}